pyvista/core/utilities/geometric_objects.py

```diff
@@ -332,16 +340,16 @@ def Sphere(
         ``start_phi`` to ``end_phi``).
 
     start_theta : float, default: 0.0
-        Starting longitude angle.
+        Starting longitude angle in degrees ``[0, 360]``.
```

Comment:
Thank you for the addition. Please also add error handling for values other than this.

Reply:
I tried to search for existing conventions on how angles are currently handled/validated in pyvista. E.g. [grid_from_sph_coords](https://github.com/pyvista/pyvista/blob/c7b29f06333d9b3427ba9ede4f30eddeef8e5c9f/pyvista/core/utilities/features.py#L123) notes the valid ranges but does no input validation, and the `prevent_seam` parameter of [texture_map_to_sphere](https://github.com/pyvista/pyvista/blob/c7b29f06333d9b3427ba9ede4f30eddeef8e5c9f/pyvista/core/filters/data_set.py#L1977C13-L1977C13) notes a range but also does no input validation. For what it's worth, `vtkSphereSource` itself will gladly accept values outside of the specified ranges and still instantiate without any issue. E.g. `start_phi=-90` will still work, and so will setting a negative radius for that matter.

So, adding error handling for `Sphere` could make it less consistent with other code and also add restrictions that are not present in the underlying VTK class. Do you still want me to add this ? Or maybe the behaviour outside these ranges could be better explained in the docstring?

Reply:
Thank you. The lack of uniformity in input validation is an issue. Let's add it after we made it in our project. I will withdraw this proposal in this Pull Request.